Vector.java

```diff
@@ -0,0 +1,41 @@
+import java.io.*;
+import java.util.*;
+import java.lang.Math; 
+
+public class Vector {
+	
+	List<Double> numbers;
+		
+	Vector (List<Double> numbers) {
+			
```

Comment:
You have a lot of empty lines.  Follow this style guide: https://google.github.io/styleguide/javaguide.html